electron/src/lib/CertificateVerifyProcManager.ts

```diff
@@ -40,6 +40,7 @@ interface DisplayCertificateErrorOptions {
 class CertificateVerifyProcManager {
   private static bypassCertificatePinning: boolean = false;
   private static isDialogLocked: boolean = false;
+  public static mainWindow: any;
```

Comment:
Please add the correct type here.
```suggestion
  public static mainWindow: Electron.BrowserWindow;
```